.github/workflows/nodejs.yml

```diff
@@ -30,10 +29,10 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 1
-      - name: Use Node.js 12.14.1
+      - name: Use Node.js 14.X.X
         uses: actions/setup-node@v2
         with:
-          node-version: 12.14.1
+          node-version: 14
```

Comment:
Should we pin the version here (current latest is `14.7.1`

Reply:
I guess if we want to ensure builds continue to work and are not affected by a bug in a Node release. Yes, let's pin.